test/scripts/hexo/multi_config_path.js

```diff
@@ -14,6 +15,22 @@ describe('config flag handling', () => {
 
   function ConsoleReader() {
     this.reader = [];
+    this.d = function(...args) {
+      var type = 'debug';
```

Comment:
If the differences of `this.d` and `this.i` are only `type`, could you refactor into a factory so that
```
this.d = readerStub("debug")
this.i = readerStub("info")
```

Thanks.

Reply:
I was just copying the style of the existing code. I didn't want to touch to many lines. It's also a test case, not production code. If you want this refactored, change impact `this.w` and `this.e` which are below these changes in the same file.

If you'd like this refactored, let me know. Otherwise I'll leave it as it.

Reply:
@JLHwung bump.